src/watch/watch.ts

```diff
@@ -1,5 +1,5 @@
 import * as path from 'path';
-import createFilter from 'rollup-pluginutils/src/createFilter';
```

Comment:
I always thought this was both a size and performance improvement but as it turns out, the new bundle is even slightly smaller, so good improvement.

Reply:
😄